### PR TITLE
Pos default empty image

### PR DESCRIPTION
**Pull request by hparfr**
Module which reduce the number of requests of product placeholder (product without image).

### CODE REVIEW

**Comment by hparfr** [comment]
CLA : false positive.

**Comment by legalsylvain** [comment]
Hi, 

@indeed CLA is [OK](https://odoo-community.org/web#id=2352&view_type=form&model=res.partner&menu_id=293&action=351). I don't know why there is a bug, do you have an idea ?

_Review_
Works like a charm. Thanks for this module.

:+1: . Code review + Test with 5000 product Database. (reactivity really improved. )

File: pos_default_empty_image/README.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
```
Comment (legalsylvain) [inline pos_default_empty_image/README.rst]: please set the same number of char as the following line.

File: pos_default_empty_image/README.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+Indeed, if the product has an image, it will load normally.
+
+Known issues
+============
+
+
+Updates
+=======
+	* Feb 2016 : First version
```
Comment (legalsylvain) [inline pos_default_empty_image/README.rst]: add empty line before  line 41 to display bullet line correctly.

File: pos_default_empty_image/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# © <2015> <Akretion, GRAP, OCA>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'POS Default empty image',
+    'version': '8.0.0.1.0',
+    'category': 'Point Of Sale',
+    'summary': 'Optimise load time for products with no image',
+    'description': """
```
Comment (legalsylvain) [inline pos_default_empty_image/__openerp__.py]: please remove the 'description' key. (duplicate with readme file)

File: pos_default_empty_image/product.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © <2015> <Akretion, GRAP, OCA>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+
+
+class ProductTemplate(models.Model):
+    _inherit = ['product.template']
+    _name = 'product.template'
```
Comment (legalsylvain) [inline pos_default_empty_image/product.py]: line 10 is unnecessary. _inherit = 'product.template' is enough.

File: pos_default_empty_image/__init__.py [hunk 5]
```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```
Comment (bealdav) [inline pos_default_empty_image/__init__.py]: imports are ascii, the default encoding, not required line

**Comment by bealdav** [comment]
:+1: 

thanks a lot

Real valuable contribution for real customer projects

File: pos_default_empty_image/__init__.py (hunk 5 above)
Comment (hparfr) [inline pos_default_empty_image/__init__.py]: It should be an empty file then ?

**Comment by hparfr** [comment]
Thanks @legalsylvain @bealdav for the review :)

**Pull request merged**